x-pack/plugins/security_solution/cypress/tasks/edit_rule.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { CustomRule } from '../objects/rule';
+import {
+  RULE_NAME_HEADER,
+  ABOUT_RULE_DESCRIPTION,
+  ABOUT_STEP,
+  ABOUT_SEVERITY,
+  ABOUT_RISK,
+  RULE_ABOUT_DETAILS_HEADER_TOGGLE,
+  INVESTIGATION_NOTES_TOGGLE,
+  ABOUT_INVESTIGATION_NOTES,
+  INVESTIGATION_NOTES_MARKDOWN,
+  DEFINITION_INDEX_PATTERNS,
+  DEFINITION_STEP,
+  DEFINITION_CUSTOM_QUERY,
+  DEFINITION_TIMELINE,
+  SCHEDULE_STEP,
+  SCHEDULE_RUNS,
+  SCHEDULE_LOOPBACK,
+} from '../screens/rule_details';
+import { EDIT_SUBMIT_BUTTON } from '../screens/edit_rule';
+
+export const expectRuleDetails = (rule: CustomRule) => {
+  const expectedTags = rule.tags.join('');
+  const expectedIndexPatterns =
+    rule.index && rule.index.length
+      ? rule.index
+      : [
```

Comment:
Since we don't seem to reference shared plugin constants within the tests, I'm wondering about the appropriate place for these kind of static, shared data.